test/workhorse/poller_test.rb

```diff
@@ -67,4 +67,54 @@ def test_nil_queues
 
     assert_equal [nil], w.poller.send(:valid_queues)
   end
+
+  def test_with_instant_repolling
+    Workhorse::DbJob.delete_all
```

Comment:
This could be moved into test teardown instead of calling it everytime when starting a new test.